openapitools.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "./node_modules/@openapitools/openapi-generator-cli/config.schema.json",
+  "spaces": 2,
+  "generator-cli": {
+    "version": "6.0.0",
+    "generators" : {
+      "python_legacy": {
+        "generatorName": "python-legacy",
+        "output": "./rest_sdk/python",
+        "glob": "./rest_sdk/openapi/openapi.json"
+      },
+      "python": {
+        "generatorName": "python",
+        "output": "./rest_sdk/python_legacy",
```

Comment:
@AliakseiButurliaa I think this is typo the output dir should match generator name but it has _legacy whereas generator does not have it.